src/gentropy/datasource/eqtl_catalogue/study_index.py

```diff
@@ -42,9 +42,10 @@ class EqtlCatalogueStudyIndex:
             StructField("sample_size", IntegerType(), True),
             StructField("quant_method", StringType(), True),
             StructField("pmid", StringType(), True),
+            StructField("study_type", StringType(), True),
         ]
     )
-    raw_studies_metadata_path = "https://raw.githubusercontent.com/eQTL-Catalogue/eQTL-Catalogue-resources/092e01a9601feb404f1c88f86311b43b907a88f6/data_tables/dataset_metadata_upcoming.tsv"
+    raw_studies_metadata_path = "https://raw.githubusercontent.com/eQTL-Catalogue/eQTL-Catalogue-resources/refs/heads/master/data_tables/dataset_metadata_upcoming.tsv"
```

Comment:
We want to commit to a fixed version of this data so that we can actually version which eQTLCatalogue data we are using.
Imagine we rerun this pipeline and we find out something is off. The problem could come from a bug we introduced, or from the raw data itself. Versioning the data is safer

Reply:
Sorry for bumping into the PR, but mayby it's better to parametrize this, so changes in external resource would not affect the code ?

Reply:
I agree. Fixing to a specific commit of the metadata is better than just using the latest but I would've thought this would've been ingested by the PIS like our other files?